### PR TITLE
Update CODEOWNERS 

### PR DESCRIPTION
**Pull request by SteveLasker**
Update CODEOWNERS to reflect the authors, editor, and active contributors for PR reviews

### CODE REVIEW

**Review by aj-stein-nist — APPROVED** [review]
I am honored but I think we need existing reviewers to endorse me to make official. 😁

**Comment by yogeshbdeshpande** [comment]
@SteveLasker : I think, this file is autogenerated ?

Could you please figure out correct way to express contribution ?

**Comment by aj-stein-nist** [comment]
> @SteveLasker : I think, this file is autogenerated ?
> 
> Could you please figure out correct way to express contribution ?

I have written a few of them and built automation around them for other non-IETF projects. tl;dr they are not auto-generated.

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners

I don't understand the question about contribution or I would try to answer it, sorry.

**Comment by SteveLasker** [comment]
- If this is a tooling question:

> I think, this file is autogenerated ?

The default template does generate on first instance, and has a make command:
```
# Automatically generated CODEOWNERS
# Regenerate with `make update-codeowners`
```

However, it does **not** auto-generate on each `make`.
In the last Architecture Editors meeting, we agreed to be more transparent about who we seek feedback using the GitHub shields for PR Reviews. 


- If you approve of A.J. being a PR reviewer please LGTM or reject with reasons.

**Comment by aj-stein-nist** [comment]
Ah, I see. If you are going to do [the relevant Makefile](https://github.com/martinthomson/i-d-template/blob/80f7f2ec576473b034965880f62eb22ce4d1bf39/setup.mk#L52) calls a Python script [that reads the draft itself after converting it xml2rfc during build](https://github.com/martinthomson/i-d-template/blob/80f7f2ec576473b034965880f62eb22ce4d1bf39/setup-codeowners.py) so adding anyone who is not author is moot. I now get what Yogesh was saying. 😆 (So much for me being "so skilled in automating these CODEOWNER files but not reading the i-d-template repo.)

I defer to you all if that is worth it, but it seems it will only look at authors, not contributors, so manually adding me to the contributor section in the draft will not work either, so up to you all if you trust me (and maybe Orie if he is already a contributor), but as built the i-d-template repo will not do that for you.

**Comment by SteveLasker** [comment]
> Ah, I see. ... so adding anyone who is not author is moot. 

As far as I can tell, that code only gets called if you explicitly call `make update-codeowners`, which would then need to be a PR which we'd reject.

This allows us to reflect additional GitHub reviewers and won't get clobbered.

**Review by OR13 — APPROVED** [review]

**Review by yogeshbdeshpande — APPROVED** [review]
LGTM!

**Pull request merged**
